packages/web-components/src/components/ic-badge/ic-badge.stories.mdx

```diff
@@ -0,0 +1,982 @@
+              slot="badge-icon"
+              xmlns="http://www.w3.org/2000/svg"
+              height="24px"
+              viewBox="0 0 24 24"
+              width="24px"
+            >
+              <path d="M0 0h24v24H0V0z" fill="none" />
+              <path
+                d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"
+              /></svg></ic-badge
+          >Large Icon</ic-button
+        >
+      </div>`}
+  </Story>
+</Canvas>
+
+### Max number
+
+<Canvas>
+  <Story name="Max number">
+    {html`<div style="padding:16px;">
+        <ic-button variant="secondary" style="margin-right:16px"
+          ><ic-badge
+            text-label="1000"
+            slot="badge"
+            size="small"
+            max-number="9"
+          ></ic-badge
+          >Small</ic-button
+        >
+        <ic-button variant="secondary" style="margin-right:16px"
+          ><ic-badge text-label="1000" max-number="9" slot="badge"></ic-badge
+          >Default</ic-button
+        >
+        <ic-button variant="secondary"
+          ><ic-badge
+            text-label="1000"
+            slot="badge"
+            size="large"
+            max-number="9"
+          ></ic-badge
+          >Large</ic-button
+        >
+      </div>
+      <div style="padding:16px;">
+        <ic-button variant="secondary" style="margin-right:16px"
+          ><ic-badge
+            text-label="1000"
+            slot="badge"
+            size="small"
+            max-number="99"
+          ></ic-badge
+          >Small</ic-button
+        >
+        <ic-button variant="secondary" style="margin-right:16px"
+          ><ic-badge text-label="1000" max-number="99" slot="badge"></ic-badge
+          >Default</ic-button
+        >
+        <ic-button variant="secondary"
+          ><ic-badge
+            text-label="1000"
+            slot="badge"
+            size="large"
+            max-number="99"
+          ></ic-badge
+          >Large</ic-button
+        >
+      </div>`}
+  </Story>
+</Canvas>
+
+### Position near
+
+<Canvas>
+  <Story name="Position near">
+    {html`<div style="padding:16px;">
+        <ic-chip label="Small" style="margin-right:16px;"
+          ><ic-badge
+            text-label="1"
+            slot="badge"
+            position="near"
+            size="small"
+          ></ic-badge
+        ></ic-chip>
+        <ic-chip label="Default" style="margin-right:16px;"
+          ><ic-badge text-label="1" slot="badge" position="near"></ic-badge
+        ></ic-chip>
+        <ic-chip label="Large"
+          ><ic-badge
+            text-label="1"
+            slot="badge"
+            position="near"
+            size="large"
+          ></ic-badge
+        ></ic-chip>
+      </div>
+      <div style="padding:16px;">
+        <ic-chip label="Small Dot" style="margin-right:16px;"
```

Comment:
I'm not sure if the positioning of the badge on the chips matches the Figma designs - e.g. the badge isn't touching the chip on this one

Also - don't know if it's possible to do this, but does the positioning also need to depend on the size of the chip? If you change the size, the badges moves closer / further away

Reply:
I think it's correct that the badge isn't touching the chip on this. From the Figma designs, it's based on the focus around the element and so from the screenshot below, it should be in the white part of the focus border into the light blue, not sitting on the element itself.
<img width="217" alt="Screenshot 2023-07-07 at 13 17 29" src="https://github.com/mi6/ic-ui-kit/assets/111882034/0f9fdc59-f0be-4e98-bee2-7f3a9a842c1f">

Not sure about the positioning depending on the size of the chip, I'd think this was up to the developer to decide which size badge to use dependent on the size of the chip, but happy to try and look into changing the positioning based on size if you think that's what should happen.